pdctl/command/region_command.go

```diff
@@ -82,6 +84,11 @@ func showRegionCommandFunc(cmd *cobra.Command, args []string) {
 		fmt.Printf("Failed to get region: %s\n", err)
 		return
 	}
+	if flag := cmd.Flag("jq"); flag != nil && flag.Value.String() != "" {
```

Comment:
can we provide a Print interface for jq or standard output?

Reply:
It's possible but I don't think it's necessary.

Reply:
seem better to extract to a function like `isJQUsed`

Reply:
There is no need to over-abstract, `--jq` is not a general parameter. these two commands just happen to both support it.